global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "3.1.100",
+    "version": "5.0.100-rc.2.20479.15",
```

Comment:
I'm not sure this is a good idea - this would switch the whole build to an rc tooling - we already had 1 bug due to this.

I guess if we leave it on `3.1.100`, then it won't be able to build `net5.0` right?

Reply:
Yes, without using a .NET 5 SDK, the `net5.0` TFM is not understood e.g. packages cannot be restored, can't build projects targeting `net5.0`, etc. The error is 

>   Microsoft.Common.CurrentVersion.targets(1180, 5): [MSB3644] The reference assemblies for .NETFramework,Version=v5.0 were not found. To resolve this, install the Developer Pack (SDK/Targeting Pack) for this framework version or retarget your application. You can download .NET Framework Developer Packs at https://aka.ms/msbuild/developerpacks

(Note it incorrectly assumes `net5.0` is a .NET Framework version)